src/core/vertexmodel.h

```diff
@@ -75,9 +85,25 @@ class VertexModel : public QStandardItemModel
     //! \copydoc mapSettings
     MapSettings *mapSettings();
 
-    Q_INVOKABLE void setGeometry( const QgsGeometry &geometry, const QgsCoordinateReferenceSystem &crs );
+    /**
+     * The coorinate reference system in which the geometry is
+     */
+    void setCrs( const QgsCoordinateReferenceSystem &crs );
 
-    Q_INVOKABLE QgsGeometry geometry() const;
+    /**
+     * The coordinate reference system in which the geometry is
+     */
+    QgsCoordinateReferenceSystem crs() const;
+
+    /**
+     * The geometry in layer coordinates
+     */
+    void setGeometry( const QgsGeometry &geometry );
+
+    /**
+     * The geometry in layer coordinates
```

Comment:
\copydoc for all 4?

Reply:
I like copydoc. sipify doesn't. That's why I stopped using it. What's your opinion?

Reply:
I don't mind, but I have been using it intensively lately.  Just that we do the same.